src/wasm-s-parser.h

```diff
@@ -135,9 +136,6 @@ class SExpressionWasmBuilder {
 
   // function parsing state
   std::unique_ptr<Function> currFunction;
-  std::map<Name, Type> currLocalTypes;
-  size_t localIndex; // params and vars
-  size_t otherIndex;
```

Comment:
`currLocalTypes` and `otherIndex` don't seem to be used anywhere. Changed `localIndex` to a local variable within `parseFunction` to reduce global status a bit.